__tests__/components/editor/InputLiteral.test.js

```diff
@@ -67,7 +67,7 @@ describe('When the user enters input into field', ()=>{
   // our mock formData function to replace the one provided by mapDispatchToProps
   const mockFormDataFn = jest.fn()
   const removeMockDataFn = jest.fn()
-  const testId = jest.spyOn(shortid, 'generate').mockReturnValue(0)
+  jest.spyOn(shortid, 'generate').mockReturnValue(0)
```

Comment:
testId was unused, so we only need the right side of the statement ...

Reply:
👍